src/Tasks/BootstrapperUtil/Util.cs

```diff
@@ -145,6 +149,44 @@ public static string GetDefaultPath(string visualStudioVersion)
             return Directory.GetCurrentDirectory();
         }
 
+        // Gets the list of additional paths to inspect for packages as defined in the registry
+        public static List<string> AdditionalPackagePaths
+        {
+            get
+            {
+                if (s_additionalPackagePaths == null)
+                {
+                    s_additionalPackagePaths = new List<string>();
```

Comment:
There shouldn't really be a race condition here because the task shouldn't be reentrant or anything, but can you accumulate this in a local variable and only assign it to the static field at the end?

Reply:
Fixed